talisker/revision.py

```diff
@@ -38,6 +39,10 @@ def version_info_txt():
         return f.read()
 
 
+def talisker_revision_id():
+    return os.environ.get('TALISKER_REVISION_ID').encode('utf-8')
```

Comment:
if there's no $TALISKER_REVISION_ID then won't this blow up?

`NoneType has no attribute 'encode'`

Reply:
Good point. I'll tidy that up.

Reply:
Fixed in https://github.com/canonical-ols/talisker/pull/267 (I hope)